### ISSUE TITLE
"No version" warning with i18next as a shared module with version >= 21.5.0

### ISSUE THREAD

**Issue by moouk** [module-federation/module-federation-examples#1248]
I have created a repository which demonstrates this issue here: https://github.com/moouk/module-federation-with-i18n-issue

When declaring i18next as shared **prior** to version 21.5.0, there was no issue.

However with version 21.5.0 onwards, the following warning occurs:

```
WARNING in shared module i18next -> C:\repos\module-federation-with-i18n-issue\app2\node_modules\i18next\dist\esm\i18next.js
No version specified and unable to automatically determine one. No version in description file (usually package.json). Add version to description file, or manually specify version in shared config.
```
Despite there being a version in packages.config, and also explicitly passing a requiredVersion, the warning still occurs.

I appreciate this might not be a module-federation issue, however do you have any advice on how to prevent this warning?

Thanks in advance!

**Comment by OriAmir** [comment]
have the same issue here @moouk 
I think it's relate to 21.5.0 release:
https://github.com/i18next/i18next/releases
![image](https://user-images.githubusercontent.com/17565646/143764643-798163b6-847e-40cd-ae67-8e285c129ff2.png)


did you find any solution ?

**Comment by OriAmir** [comment]
it's fixed on `21.5.4` : https://github.com/i18next/i18next/issues/1699
I think it's can be closed.
